demos/transport/tls_utils.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2020 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+/**
+ * @file tls_utils.h
+ * @brief Wrapper utilities for using OpenSSL based TLS stack for demos and
+ * integration tests.
+ */
+
+#ifndef TLS_UTILS_H_
+#define TLS_UTILS_H_
+
+/* OpenSSL include. */
+#include <openssl/ssl.h>
+
+#include "mqtt.h"
+
+/**
+ * @brief Creates a TCP connection to the MQTT broker as specified by
+ * BROKER_ENDPOINT and BROKER_PORT defined at the top of this file.
```

Comment:
There is no BROKER_ENDPOINT and BROKER_PORT at the top of this file.

Reply:
Fixed